.changeset/happy-singers-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-fetch": minor
+---
+
+feat: set baseUrl per request
```

Comment:
Could be a bit more explicit as this will be used in the changelog:
```suggestion
Allow specifying baseUrl per request
```

Reply:
Sure, good point.